warnings running tests

<!--- Provide a brief summary of the issue -->
Warnings running `mvn test`:
```
[INFO] Running com.oracle.javafx.scenebuilder.kit.skeleton.SkeletonBufferKotlinTest
[ERROR] WARNING: A restricted method in java.lang.System has been called
[ERROR] WARNING: java.lang.System::load has been called by com.sun.glass.utils.NativeLibLoader in module javafx.graphics (file:/Users/JosePereda/.m2/repository/org/openjfx/javafx-graphics/24.0.1/javafx-graphics-24.0.1-mac-aarch64.jar)
[ERROR] WARNING: Use --enable-native-access=javafx.graphics to avoid a warning for callers in this module
[ERROR] WARNING: Restricted methods will be blocked in a future release unless native access is enabled
[ERROR] 
```

### Expected Behavior
<!--- Tell us what should happen -->

### Current Behavior
<!--- Tell us what happens instead of the expected behavior -->

### Steps to Reproduce
<!--- Steps to reproduce this bug -->

### Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->

### Screenshots
<!--- If applicable, add screenshots to help explain the bug report -->

--- a/pom.xml
+++ b/pom.xml
@@ -142,7 +142,7 @@
                     <configuration>
                         <reuseForks>false</reuseForks>
                         <forkCount>1</forkCount>
-                        <argLine>--add-opens=javafx.fxml/javafx.fxml=com.gluonhq.scenebuilder.kit -Djavafx.allowjs=true</argLine>
+                        <argLine>--add-opens=javafx.fxml/javafx.fxml=com.gluonhq.scenebuilder.kit -Djavafx.allowjs=true --enable-native-access=javafx.graphics</argLine>
                     </configuration>
                 </plugin>
             </plugins>

